### PR TITLE
Creating Training Pages for GCC Training Sessions

### PR DESCRIPTION
**Pull request by assuntad23**
I updated the training page to have a cards view (thanks @afgane for the headstart!) , created a page for each training session, and added training back into the header, now that it will be populated with content. 

Each training has it's own page, with a training abstract and bios for the instructors. Some were not provided yet and instead have a banner that says more info will be coming soon. I will create additional PRs as I get this information, but for now, this is good to go.

### CODE REVIEW

**Comment by hechth** [comment]
@assuntad23 is this ready to be merged or do you still want to collate more information in this PR?

**Comment by assuntad23** [comment]
@hechth still adding more info to each page, not ready yet.

File: content/events/gcc2024/training/backend-dev/index.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+### John Chilton
+Penn State University <br>
+Galaxy Developer
+
+### David Lopez
+University of Freiburg <br>
+I'm David López, a Software Developer working as part of the European Galaxy Team at the University of Freiburg. Over the past three years, I've been involved full-time in the development and maintenance of the Galaxy code. This experience has provided me with in-depth insights into the intricacies of Galaxy's functionality.
+
+With this knowledge, my goal is to empower each of you to contribute effectively to the ongoing evolution of the Galaxy project. Together, we can work towards improving a platform that computational research available for everyone.
```
Comment (davelopez) [inline content/events/gcc2024/training/backend-dev/index.md:24]: I guess the Bio from the Trainers Form was completely out of place as I can see :laughing: 
```suggestion
Galaxy Developer
```

**Review by tcollins2011 — APPROVED** [review]
I think its looking good. I'll try and harass people to get the rest of the abstract info.

**Comment by assuntad23** [comment]
^ Sounds good @tcollins2011 . I think we should merge this as is for now, and update other pages as the info comes in.

**Comment by afgane** [comment]
> ^ Sounds good @tcollins2011 . I think we should merge this as is for now, and update other pages as the info comes in.

Give me a bit of time please - I'd like to update a few boxes.

**Pull request merged**